app/dashboard/notifications.py

```diff
@@ -209,7 +220,7 @@ def build_github_notification(bounty, event_name, profile_pairs=None):
     from dashboard.models import BountyFulfillment
 
     msg = ''
-    usdt_value = f"({round(bounty.value_in_usdt, 2)} USD)" if bounty.value_in_usdt else ""
+    usdt_value = "(" + str(round(bounty.value_in_usdt, 2)) + " USD @ " + str(round(convert_token_to_usdt(bounty.token_name))) + " USD/" + bounty.token_name + ")" if bounty.value_in_usdt else ""
```

Comment:
Any reason you migrated away from literal string interpolation here?

Reply:
Oh, interesting. I think that might have been a merge issue because I just kept the format that was there. I will update to literal string interpolation.